andela/kingsmen-ah-backend
# #166415129 feat database schema

#### What does this PR do?
This PR designs the entire database schema for the application

#### Description of Task to be completed?
- create migrations
- create models
- associate each table

#### How should this be manually tested?
- run `npm run refreshdb`, should see a couple of tables being migrated in the database

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#166415129](https://www.pivotaltracker.com/story/show/166415129)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
